app/models/letter_opener_web/letter.rb

```diff
@@ -77,6 +79,14 @@ def style_exists?(style)
       File.exist?("#{base_dir}/#{style}.html")
     end
 
+    def exists?
+      File.exist?(base_dir)
```

Comment:
thought: As `#base_dir` is actually an instance of `Pathname`, we could also change this to:

```suggestion
      base_dir.exist?
```

But I wanted to keep the diff as small as possible.